spec/usermanual/customcourse.html.erb

```diff
@@ -278,6 +286,14 @@ require 'cgi' # for escapeHTML
   <h2>Personalized course progression</h2>
 
   <p>
+    <i>
+      Notice that course progression can now be configured from the webapp.
+      Configuration from the repository is only loaded initially, and will not change
+      after subsequent refreshes.
+    </i>
```

Comment:
teacher_sets_deadlines_spec: scenario 'Refreshing course does not overwrite deadlines set in the form'
Works for all courses at the moment.
I'm not entirely sure how we could provide setting deadlines through repository and webapp simultaneously